### PR TITLE
Make `DEFAULT_MAX_MCCABE_COMPLEXITY` configurable via flake8 kwarg and setup.py

### LINKED ISSUE #2: Make `DEFAULT_MAX_MCCABE_COMPLEXITY` configurable via flake8 kwarg and setup.py

### PR DESCRIPTION
**Pull request by lowitea**
closes #2

### CODE REVIEW

File: flake8_adjustable_complexity/checker.py [hunk 1]
```diff
@@ -68,3 +68,17 @@ def run(self) -> Generator[Tuple[int, int, str, type], None, None]:
...
+            '--max-mccabe-complexity',
+            type=int,
+            parse_from_config=True,
+            help='Max mccabe complexity',
+            default=cls.DEFAULT_MAX_MCCABE_COMPLEXITY,
+        )
+
+    @classmethod
+    def parse_options(cls, options) -> None:
+        cls.DEFAULT_MAX_MCCABE_COMPLEXITY = int(options.max_mccabe_complexity)
```
Comment (Melevir) [inline flake8_adjustable_complexity/checker.py]: `cls.DEFAULT_MAX_MCCABE_COMPLEXITY` is wrong name now. It should be not upper-case and without "default" prefix (`cls.max_mccabe_complexity`).

File: flake8_adjustable_complexity/checker.py [hunk 2]
```diff
@@ -68,3 +68,17 @@ def run(self) -> Generator[Tuple[int, int, str, type], None, None]:
                 self._error_message_template.format(actual_complexity, max_expected_complexity),
                 type(self),
             )
+
+    @classmethod
+    def add_options(cls, parser) -> None:
+        parser.add_option(
+            '--max-mccabe-complexity',
```
Comment (Melevir) [inline flake8_adjustable_complexity/checker.py:77]: Also, info about the argument should be added to readme.

Comment (lowitea) [reply flake8_adjustable_complexity/checker.py:77]: Done)

File: flake8_adjustable_complexity/checker.py (hunk 1 above)
Comment (lowitea) [reply flake8_adjustable_complexity/checker.py]: Yes. All right :)

**Comment by Melevir** [comment]
Thanks!

**Pull request merged**